themes/SuiteP/css/suitep-base/email.scss

```diff
@@ -163,7 +163,7 @@
       outline: none;
     }
 
-    .vertical-separator {
+    .vertical-seperator {
```

Comment:
You'd need to revert this for BC or keep it and have a duplicate class with the old name.